src/components/timeline/timeline.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, AfterViewInit, ElementRef, Renderer2 } from '@angular/core';
+
+@Component({
+    selector: 'ux-timeline',
+    templateUrl: './timeline.component.html'
+})
+export class TimelineComponent implements AfterViewInit {
+
+    constructor (

```

Comment:
To add a class to a host element we can use the `host` property in the component decorator. You shouldn't need to use the renderer or elementRef

Reply:
Renderer no longer used. The 'side' attribute is no longer used.